caom2pipe/tests/test_data_source_composable.py

```diff
@@ -326,14 +326,14 @@ def test_list_dir_separate_data_source(test_config):
     assert test_subject is not None, 'ctor is broken'
     test_result = test_subject.get_work()
     assert test_result is not None, 'expect a result'
-    assert len(test_result) == 101, 'expect contents in the result'
+    assert len(test_result) == 100, 'expect contents in the result'
```

Comment:
Why did these numbers change?

Reply:
Because I bzip2-compressed one of the test files in the directory for a different test, and that extension is excluded by the test case.